api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
There may be a problem here. We have customers who are already using the `memoryused` parameter value. We cannot just remove or change that to `memoryutil`, which will be considered a regression. We have to provide a deprecation path if we want to move to `memoryutil` from `memoryused`.